pkg/apis/flowcontrol/install/install.go

```diff
@@ -38,5 +39,9 @@ func Install(scheme *runtime.Scheme) {
 	utilruntime.Must(flowcontrolv1alpha1.AddToScheme(scheme))
 	utilruntime.Must(flowcontrolv1beta1.AddToScheme(scheme))
 	utilruntime.Must(flowcontrolv1beta2.AddToScheme(scheme))
-	utilruntime.Must(scheme.SetVersionPriority(flowcontrolv1beta1.SchemeGroupVersion, flowcontrolv1beta2.SchemeGroupVersion, flowcontrolv1alpha1.SchemeGroupVersion))
+	utilruntime.Must(flowcontrolv1beta3.AddToScheme(scheme))
+	// TODO: This controls serialization order, do we still need to
+	//  serialize to v1beta1 since it is slated to be removed in 1.26?
+	utilruntime.Must(scheme.SetVersionPriority(flowcontrolv1beta1.SchemeGroupVersion, flowcontrolv1beta2.SchemeGroupVersion,
```

Comment:
Questions:
- if we remove `v1beta1`, that implies `v1alpha1` can also be removed, right?
- after removing `v1beta1` and `v1alpha1`, the serialization order will be `v1beta2, v1beta3` (1.26), and in 1.27 we will change it to `v1beta3, v1beta2`?

I am planning to address removal of `v1beta1` in a separate PR.

Reply:
we need to keep all serializations available because we do not require storage version migration.  We can now make the serialization version be v1beta2 because vN-1 understands that level.